src/README.md

```diff
@@ -45,6 +45,28 @@ spring.datasource.password=postgres
 ./gradlew -x build -x test
 ```
 
+### Enable Https forward
+
+First, please obtain the client certificate file according to this [document](http://pulsar.apache.org/docs/en/security-tls-transport/). We use the following command to generate the keystore file for Spring Boot.
+
+```$xslt
+keytool -import -trustcacerts -keystore keystore-file -alias test-keystore -file certs/ca.cert.pem
+```
+
```

Comment:
Thirdly, configure Pulsar Manager to enable TLS by adding the following configuration settings.